### ISSUE TITLE
installation fails on Windows with MSYS2

### ISSUE THREAD

**Issue by vtorri** [libressl-portable/portable#429]
Hello

on Windows, with MSYS2, the installation of libressl 2.7.3 fails : 

ln: impossible to create symbolic link 'E:/Documents/MSYS2/home/vtorri/opt_ewpi/share/man/man3/OCSP_crlID_new.3': No such file or directory

I have no idea why.

**Comment by normanmaurer** [comment]
Same here on macOS and Linux...

**Comment by busterb** [comment]
Sounds like a symptom of having two case-differentiated manpages on a case-insensitive filesystem.

Can you all verify that your target filesystem is case-insensitive?

**Comment by vtorri** [comment]
NTFS is case-insensitive, FOO and foo are the same

**Comment by busterb** [comment]
OK. This could be a different issue entirely, checking it out now. Assuming everyone is using './configure' and not cmake?

**Comment by busterb** [comment]
Yep, the filesystem doesn't support linking a file to a name with a different case:

ln -sf "OCSP_CRLID_new.3" "$(DESTDIR)$(mandir)/man3/OCSP_crlID_new.3"

Curious it doesn't trigger an error on my case-insensitive Mac (using the new APFS), but instead overwrites the file with a symlink:

```
bcook@mini:/usr/local/share/man/man3$ ls -l OCSP_crlID_new.3 
lrwxr-xr-x  1 root  admin  16 Jun 15 08:24 OCSP_crlID_new.3 -> OCSP_CRLID_new.3
bcook@mini:/usr/local/share/man/man3$ cat OCSP_crlID_new.3 
cat: OCSP_crlID_new.3: Too many levels of symbolic links
```

**Comment by busterb** [comment]
Fixed in b78f2939398db49bb1dfabaa48529d11ae687e20
